src/visualizers/Turtle.ts

```diff
@@ -167,4 +161,4 @@ class Turtle extends P5Visualizer {
     }
 }
 
-export const exportModule = new VisualizerExportModule('Turtle', Turtle, '')
+export const exportModule = new VisualizerExportModule(Turtle)
```

Comment:
maybe 'Terms of the sequence control a colourful walk on the screen.' or 'Turtle walks according to terms in the sequence, creating a trail.'